ansibledoctor/templates/readme/_vars_tabulated.j2

```diff
@@ -0,0 +1,36 @@
+{% set var = role.var | default({}) %}
+{% if var %}
+## Default Variables
+
+{%     set columns = ["variable", "default", "description", "type", "deprecated", "example"] %}
+{%     set found_columns = ["variable", "default"] + var.values() | map("list") | sum(start=["key"]) | unique | list %}
+{%     for c in columns %}
+{%         if c in found_columns %}
+| {{ c | capitalize }} {# trimnewline #}
```

Comment:
In order to make the jinja2 template readable, I tried to keep each column value on a different line and let jinja collapse them back together. Without the `{# trimnewline #}` there (or any `{# foobar #}` type block), newlines would persist after each column value and make the markdown table render incorrectly.

This is taking advantage of jinja2's default removal of whitespace around blocks. The same functionality could have been achieved by putting `{% if true %}{% endif %}` but I figured that `{# trimnewline #}` better communicates its purpose.

Reply:
Understood. I think that's also because of the `jinja-statements-delimiter` rule from j2lint. However, even after reading https://github.com/aristanetworks/j2lint/issues/53 it sounds very opinionated to me and I don't really see why `{%-` or `{%+` should not be used. In this particular case, we could just use `{{ c | capitalize -}}. To get forward, I'll merge the PR as is, but I might change it afterward. Please let me know if there is a real benefit of not using these jinja2 delimiters.